src/audit_simulation_sarah.py

```diff
@@ -0,0 +1,155 @@
+from src.multivariate_hypergeom_sample import multivhyper
+from src.brla_sarah import brla
+import numpy as np
+
+"""
+Class for simulating Bayesian RLA.
+Can be used on 2-candidate or multi-candidate elections with or without invalid votes.
+"""
```

Comment:
Block comments should be the first object in the file (coming only perhaps after a #! line)
Then they are used as documentation by pydoc etc.